src/App.js

```diff
@@ -12,6 +12,7 @@ import SignUp from './components/auth/SignUp'
 import SignIn from './components/auth/SignIn'
 import SignOut from './components/auth/SignOut'
 import ChangePassword from './components/auth/ChangePassword'
+import IndexDepartment from './components/department/indexDepartment'
```

Comment:
make sure that your import matches the file name exactly! so you have department lowercase, and IndexDepartment camel case. above you would write 
`import IndexDepartment from './components/department/IndexDepartment'` making sure IndexDepartment is camel case

Reply:
Also if you import your component into App.js, I think you want to add it to the page as well. so before the end of `<Fragment />` in the return, you could put `<IndexDepartment/>`, for example

Reply:
Also if you import your component into App.js, I think you want to add it to the page as well. so before the end of `<Fragment /> (line 82) in the return, you could put <IndexDepartment/>, for example